projects/e2e-tests/codecov_tests/src/lib.rs

```diff
@@ -281,39 +281,35 @@ pub extern "C" fn finish() -> i32 {
                 amount.len(),
             )
         },
-        19,
+        62,
         "trace_amount",
     );
 
     // ########################################
     // Step #2: Test set_data edge cases
     // ########################################
-    check_result(
-        unsafe { host_bindings_loose::get_parent_ledger_hash(-1, 4) },
```

Comment:
Passing a negative number to `get_parent_ledger_hash` is not impossible per Rust type-checking.